projects/packages/forms/src/blocks/field-name/variations.js

```diff
@@ -67,9 +78,10 @@ const variations = [
 		description: __( 'Collect the visitor’s last name.', 'jetpack-forms' ),
 		icon,
 		scope: [ 'inserter', 'transform' ],
-		isActive: [ 'id' ],
+		isActive: blockAttributes => blockAttributes.fieldVariant === LAST_NAME_ID,
```

Comment:
We now determine if a variant is active based on the fieldVariant attribute for all variants.